examples_gbs/run_tutorial_similarity.py

```diff
@@ -0,0 +1,278 @@
+# Creating a feature vector
+# -------------------------
+#
+# Following :cite:`schuld2019quantum`, we can create a *feature vector* to describe each graph.
+# These feature vectors contain information about the graphs and can be viewed as a mapping to a
+# high dimensional feature space, a technique often used in machine learning that allows us to
+# find ways of separating points in the space for classification.
+#
+# The feature vector of a graph can be composed in a variety of ways. One approach is to built it
+# using statistics from a GBS device configured to sample from the graph, as we now discuss.
+#
+# We begin by defining the concept of an *orbit*, which contains all the GBS samples that are
+# equivalent under permutation of the modes. A sample can be converted to its corresponding orbit
+# using the :func:`~.sample_to_orbit` function. For example, the first sample of ``m0`` is ``[0,
+# 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0]`` and has orbit:
+
+print(similarity.sample_to_orbit(m0[0]))
+
+##############################################################################
+# Other samples can be randomly generated from the ``[1, 1]`` orbit using:
+
+print(similarity.orbit_to_sample([1, 1], modes=m0.modes))
+
+##############################################################################
+# Orbits provide a useful way to coarse grain the samples from GBS into outcomes that are
+# statistically more likely to be observed. However, we are interested in coarse graining further
+# into **events**, which correspond to a combination of orbits with the same photon number such
+# that the number of photons counted in each mode does not exceed a fixed value
+# ``max_count_per_mode``. To understand this, let's look at all of the orbits with a photon
+# number of 5:
+
+print(list(similarity.orbits(5)))
+
+##############################################################################
+# All 5-photon samples fall into one of the orbits above. A 5-photon event with
+# ``max_count_per_mode = 3`` means that we include the orbits: ``[[1, 1, 1, 1, 1], [2, 1, 1, 1],
+# [3, 1, 1], [2, 2, 1], [3, 2]]`` and ignore the orbits ``[[4, 1], [5]]``. For example,
+# the sample ``[0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 3, 0, 0, 0]`` is a 5-photon event:
+
+print(similarity.sample_to_event([0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 3, 0, 0, 0], 3))
+
+##############################################################################
+# Samples with more than ``max_count_per_mode`` in any mode are not counted as part of the event:
+
+print(similarity.sample_to_event([0, 4, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0], 3))
+
+##############################################################################
+# Now that we have mastered orbits and events, how can we make a feature vector? It was shown in
+# :cite:`schuld2019quantum` that one way of making a feature vector of a graph is to measure the

```

Comment:
I'm not sure "measure" is the right word here. Maybe "obtain"? "calculate"? "estimate"?

Reply:
Fixed